Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
Honestly, I can't be bothered to continue an argument a day later Mike. I've just taken up running my own fork as a primary, it's easier than this.